include/seccomp-syscalls.h

```diff
@@ -273,6 +273,7 @@
 #define __PNR_timerfd_settime64			-10239
 #define __PNR_utimensat_time64			-10240
 #define __PNR_ppoll				-10241
+#define __PNR_openat2				-10242
```

Comment:
A `__PNR_xxx` define isn't needed as `openat2` is defined for every ABI we currently support.